jekyll/_cci2/ip-ranges.md

```diff
@@ -100,13 +100,17 @@ IP address ranges for core services (used to trigger jobs, exchange information
 
 - 18.214.70.5
 - 52.20.166.242
-- 35.174.249.131
 - 18.214.156.84
 - 54.236.156.101
+- 52.22.215.219
+- 52.206.105.184
+- 52.6.77.249 
+- 34.197.216.176
+- 35.174.249.131
 - 3.210.128.175
 
-**Upcoming Changes to list of IP address ranges** (Last Updated: 08/02/2021):
-None
+**Upcoming Changes to list of IP address ranges** (Last Updated: 08/23/2021):
+Added new IP address ranges to list of IP address ranges for core services on 08/23.  Removed 35.174.249.131 from list of IP address ranges for core services on 08/23. Machine-consumable lists have been updated to reflect new IP address ranges.
```

Comment:
Please double-check that the IP address  `35.174.249.131` should be listed as removed -- it looks like it was added above, and it is also added to the A-records in the other pull request.

I'm also including this format as a suggestion. If we make each date an h4, we can easily create a timeline of changes as time goes on.

```suggestion
#### 2021-08-23
* Added new items to the list of IP address ranges for core services.
* Removed `35.174.249.131` from the list of IP address ranges for core services.

The machine-consumable lists have also been updated to reflect the new IP address ranges.
```

Reply:
Correct, `35.174.249.131` should _not_ be removed.